packages/core-data/src/queried-data/get-query-parts.js

```diff
@@ -41,6 +41,7 @@ export function getQueryParts( query ) {
 		perPage: 10,
 		fields: null,
 		include: null,
+		context: 'default',
```

Comment:
If the `context` query param is omitted, the default value is `view`.

Reply:
no, we do enforce "edit" context in the API requests, even if the context is "view" by default for the server, that's why I opted on a new "default"/"empty" context in the frontend. Backward compatibility.

Reply:
Does that mean that calling `getQueriedItems` without `context` will issue an API request with `?context=edit` query param and will store the result under the `state[ 'default' ]` key?

And that `getQueriedItems( state, { context: 'edit' } )` will issue a request with the same `?context=edit` query param but this time will store the result as `state[ 'edit' ]`?

Reply:
> Does that mean that calling getQueriedItems without context will issue an API request with ?context=edit query param and will store the result under the state[ 'default' ] key?

In most entities yes, but entities can define another set of default REST API parameters. 

I agree it's not idea that `edit` is different than `default` in this cases, but solving it properly would have required a small change in the entities API (explicitly define a default context instead of relying on a more global baseURLParams). I think it's something we can try but I thought it was not important for v1 of this.

Reply:
I agree that it's not terribly important. I mostly wanted to confirm that this little flaw is really there, and that it's not just my misunderstanding.